### ISSUE TITLE
[padding-line-between-statements] Support multiline and singleline imports

### ISSUE THREAD

**Issue by Ky6uk** [eslint/eslint#11626]
**Please describe what the rule should do:**

Add an option `multiline-import` and `singleline-import` to rule `padding-line-between-statements`.

**What category of rule is this? (place an "X" next to just one item)**

- [X] Enforces code style (layout)

**Provide 2-3 code examples that this rule will warn about:**

```js
import Woof from 'woof';

import { 
  Bork,
  Bark,
  Brork
} from 'bork';
```

**Comment by platinumazure** [comment]
Hi @Ky6uk, thanks for the issue.

I think I see what you're requesting, but just to make sure anyone who sees this issue understands, could you please provide some more fleshed-out examples? (The goal is to show examples where the rule currently doesn't warn but it would with the new option, and/or cases where the rule currently does warn but it would with a new option. You could probably contract `multiline-import` and `singleline-import` with plain old `import`, I suppose.)

**Comment by Ky6uk** [comment]
This rule is working absolutely identical as `multiline-const` and `singleline-const` but for imports. It means there are no differences between "plain old import" (what is this? 🤔) and other kinds of imports.

```js
// import on a single line (singleline-import)
import { A, B, C } from './module';

// import on few lines (multiline-import)
import { 
  A, 
  B, 
  C 
} from './module';
```

So, for example, if I prefer to stack single line imports together and to add an extra line around multiline imports, I will use the next:

```json
{
  "padding-line-between-statements": ["error", {
      "blankLine": "always",
      "prev": ["import"],
      "next": "*"
    }, {
      "blankLine": "always",
      "prev": "*",
      "next": ["import"]
    }, {
      "blankLine": "never",
      "prev": ["singleline-import"],
      "next": ["singleline-import"]
    }]
}
```

**Comment by platinumazure** [comment]
Thanks @Ky6uk. I'm :+1: for adding this.
